src/mrack/outputs/pytest_multihost.py

```diff
@@ -125,14 +125,18 @@ def create_multihost_config(self):  # pylint: disable=too-many-branches
                     mhcfg["ad_sub_hostname"] = host["name"]
                     mhcfg["ad_sub_ip"] = host["ip"]
 
-                host_custom_attrs = host.get("pytest_multihost")
+                glob_custom_attrs = self._metadata.get("pytest_multihost", {})
+                domain_custom_attrs = domain.get("pytest_multihost", {})
```

Comment:
Trivial detail, but here "global" is abbreviated  to `glob`, while "domain" is not. However, "domain" was abbreviated to `dom` in `ansible_inventory.py`. It would be more coherent to use the same abbreviation here.

Reply:
fixed